apps/src/lib/cli.rs

```diff
@@ -6936,6 +6963,32 @@ pub mod args {
         }
     }
 
+    #[derive(Clone, Debug)]
+    pub struct TestGenesis {
+        /// Templates dir
+        pub path: PathBuf,
+        pub wasm_dir: PathBuf,
+    }
+
+    impl Args for TestGenesis {
+        fn parse(matches: &ArgMatches) -> Self {
+            let path = PATH.parse(matches);
+            let wasm_dir = WASM_DIR.parse(matches).unwrap_or_default();
+            Self { path, wasm_dir }
+        }
+
+        fn def(app: App) -> App {
+            app.arg(
+                PATH.def()
+                    .help("Path to the directory with the template files."),
+            )
+            .arg(WASM_DIR.def().help(
+                "Can optionally provide a wasm directory as part of verifying \
```

Comment:
nit: the wording here should be more consistent with the rest of the CLI subcmd help messages. something along the lines of "Optional wasm directory to blah blah ..."